### DISCUSSION TITLE
Can I move the location of the database?

### DISCUSSION THREAD

**Discussion by agglomerator** [vzhd1701/evernote-backup#58] [General]
My database is 19GB and was created in "c:\users\myname' and would like to put it on another drive.  Can I move it after it is created, or is there an argument I can pass that will cause it to be created on another location?

**Comment by vzhd1701** [comment]
Yes, you can move the database wherever you like, you can set the database file location with `--database` option like so:
```bash
$ evernote-backup sync --database "D:\some_dir\database.db"
```
